sprokit/processes/core/video_input_process.cxx

```diff
@@ -237,10 +285,22 @@ ::_step()
       }
     }
 
+    kwiver::vital::path_t filename = d->m_video_reader->filename();
+
+    if ( d->m_no_path_in_name )
+    {
+      const size_t last_slash_idx = filename.find_last_of("\\/");
```

Comment:
This is wrong on POSIX systems (a file name may contain '\\'). Please use a portable definition of the path separator character.